R/data.R

```diff
@@ -4,3 +4,16 @@
 #' @keywords datasets
 #' @family datasets
 "dm_neuro"
+
+
```

Comment:
Could you please remove these two empty lines

Reply:
Done!